openfisca_canada_mvohwr/variables/motor_vehicle_operator_hours_of_work_regulation/mvo_driver_classification.py

```diff
@@ -0,0 +1,126 @@
+from openfisca_core.model_api import *
+from openfisca_canada_mvohwr.entities import Person
+
+class mvo_driver_classification_type(Enum):
+    city_motor_vehicle_operator = u'City Motor Vehicle Operator'
+    highway_motor_vehicle_operator = u'Highway Motor Vehicle Operator'
+    bus_operator = u'Bus Operator'
+    other = u'Other'
+
+class mvo__driver_classification(Variable):
+    value_type = Enum
+    possible_values = mvo_driver_classification_type
+    default_value = mvo_driver_classification_type.other
+    entity = Person
+    label = u""
+    definition_period = DAY
+    reference = u"TODO"
+
+    def formula(persons, period, parameters):
+        return select (
+        [
+            persons("mvo__is_cmvo",period),
+            persons("mvo__is_hmvo",period),
+            persons("mvo__is_bus_operator",period)
+        ],
+        [
+            mvo_driver_classification_type.cmvo,
+            mvo_driver_classification_type.hmvo,
+            mvo_driver_classification_type.bus_operator,
+        ],
+        mvo_driver_classification_type.other
+        )
+
+class mvo__is_cmvo(Variable):
+    value_type = bool
+    entity = Person
+    label = u"TODO"
+    definition_period = DAY
+    reference = u"TODO"
+    
+    def formula(persons, period, parameters):
+        return persons("mvo__is_motor_vehicle_operator",period) &\
+            not_(persons("mvo__is_bus_operator",period)) &\
+            (persons("mvo__is_classified_as_cmvo",period) + persons("mvo__operates_close_to_home_terminal",period))
+
+class mvo__is_hmvo(Variable):
+    value_type = bool
+    entity = Person
+    label = u"TODO"
+    definition_period = DAY
+    reference = u"TODO"
+
+    def formula(persons, period, parameters):
+        return persons("mvo__is_motor_vehicle_operator",period) &\
+            not_(persons("mvo__is_bus_operator",period)) &\
+            not_(persons("mvo__is_cmvo",period))
```

Comment:
Is the default someone is a highway driver if they drive a motor vehicle and nothing else?

Reply:
This is the issue we discussed with Pia.  According to the regulations the definition of a highway operator is defined has anyone that operates a motorized vehicle that is not a city or bus operator.

Reply:
Sounds good!